makefile.shade

```diff
@@ -206,9 +209,9 @@ var RUNTIME_TARGETS='${new List<RuntimeTarget>()}'
         }
     }
 
-#build-dnx-win .ensure-msbuild .update-rc-files target='build-windows'
-    var bootstrapperProj = '${Path.Combine(ROOT, "src", BOOTSTRAPPER_EXE_NAME, BOOTSTRAPPER_EXE_NAME + ".vcxproj")}'
-    @{
+#build-dnx-win-exe .ensure-msbuild .update-rc-files target='build-windows'
+    var bootstrapperProj = '${Path.Combine(ROOT, "src", BOOTSTRAPPER_WINDOWS_FOLDER_NAME, "dnx.windows.vcxproj")}'
```

Comment:
Not sure. It is for folder name and that's why I did not want to use it. The assumption that the project name and the folder name are the same does not have to be correct.